doc/python/static-image-export.md

```diff
@@ -41,31 +41,20 @@ Plotly figures are interactive when viewed in a web browser: you can hover over
 
 
 <!-- #region -->
-#### Install Dependencies
-Static image generation requires the [orca](https://github.com/plotly/orca) commandline utility and the [psutil](https://github.com/giampaolo/psutil) and [requests](https://2.python-requests.org/en/master/) Python libraries. There are 3 general approach to installing these dependencies.
-
-##### conda
-Using the [conda](https://conda.io/docs/) package manager, you can install these dependencies in a single command:
+#### Install Dependencies (Kaleido)
+Static image generation requires the [`kaleido`](https://github.com/plotly/Kaleido) package which can be installed using pip...
 ```
-$ conda install -c plotly plotly-orca==1.2.1 psutil requests
+$ pip install -U kaleido
 ```
 
-**Note:** Even if you do not want to use conda to manage your Python dependencies, it is still useful as a cross platform tool for managing native libraries and command-line utilities (e.g. git, wget, graphviz, boost, gcc, nodejs, cairo, etc.).  For this use-case, start with [Miniconda](https://conda.io/miniconda.html) (~60MB) and tell the installer to add itself to your system `PATH`.  Then run `conda install plotly-orca==1.2.1` and the orca executable will be available system wide.
-
-##### npm + pip
-You can use the [npm](https://www.npmjs.com/get-npm) package manager to install `orca` (and its `electron` dependency), and then use pip to install `psutil`:
-
-```
-$ npm install -g electron@1.8.4 orca
-$ pip install psutil requests
+or conda.
 ```
+$ conda install -c plotly python-kaleido
+``` 
 
-##### Standalone Binaries + pip
-If you are unable to install conda or npm, you can install orca as a precompiled binary for your operating system. Follow the instructions in the orca [README](https://github.com/plotly/orca) to install orca and add it to your system `PATH`. Then use pip to install `psutil`.
+#### Install Dependencies (Orca)
+While Kaleido is now the recommended approach, image export can also be supported by the [orca](https://github.com/plotly/orca) command line utility. See the [Orca Management](/python/orca-management/) section for instructions on installing, configuring, and troubleshooting orca.
 
-```
-$ pip install psutil requests
-```
 <!-- #endregion -->
 
 ### Create a Figure
```

Comment:
line 61 should be "Now let's create a simple scatter plot with 100 random points of varying color and size."

Reply:
Fixed in 70c11ac67c02f4fc0cbe4003f99b75d0941c2ab1